packages/pages/src/common/src/function/internal/types.test.ts

```diff
@@ -0,0 +1,141 @@
+import { convertFunctionModuleToFunctionModuleInternal } from "./types.js";
+import { ServerlessFunction, FunctionModule } from "../types.js";
+
+const exampleFunction: ServerlessFunction = () => {
+  return { body: "Hello World", headers: {}, statusCode: 200 };
+};
+
+// JSON.stringify does not include functions in the output string
+// This function adds the function's name in place of the function
+const stringifyIncludeFunctionNames = (key: string, value: any) => {
+  if (typeof value === "function") {
+    return value.name;
+  } else {
+    return value;
+  }
+};
+
+describe("internal/types - convertFunctionModuleToFunctionModuleInternal", () => {
+  it("converts a function in /function/http using the default getPath and config", async () => {
+    const functionModule: FunctionModule = {
+      default: exampleFunction,
+    };
+    const functionModuleInternal =
+      convertFunctionModuleToFunctionModuleInternal(
+        "src/functions/http/api/example.ts",
+        functionModule,
+        false
+      );
+    const expected = {
+      default: "exampleFunction",
```

Comment:
This shouldn't be a string.

Reply:
The tests I copied from features were comparing with JSON.stringify, which doesn't output functions.
I used a modifier to replace the function with it's name for testing but I can look on Monday if there's a better way to test it